### PR TITLE
feat: steps by sanity

### PR DESCRIPTION
**Pull request by Fedeya**
CU-1w988ex

### make more dynamic the website with the sanity, can change the steps in the sanity

# Changes
- changed type of page adding the step array in `steps` field
- created component for steps called `Steps` and removed `MentorshipsSteps` 
- changed query of get pages by hero

### CODE REVIEW

**Comment by hifrontendcafe** [comment]
Task linked: [CU-1w988ex [Mentorías] Crear componente de pasos para cargar los datos](https://app.clickup.com/t/1w988ex)

File: src/components/Steps/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  steps: Step[];
+};
+
+const Steps: React.FC<StepsProps> = ({ steps }) => {
+  return (
+    <section className="text-gray-100 body-font">
+      <div className="px-5 py-32 lg:px-0">
+        <div className="flex flex-wrap -mx-4 -mt-4 -mb-10 sm:-m-4">
+          {steps
+            .sort((a, b) => (a > b ? -1 : 1))
```
Comment (SebasEC96) [inline src/components/Steps/index.tsx:13]: Does this work? Shouldn't it be like this?
```suggestion
            .sort((a, b) => (a.step - b.step))
```

Comment (Fedeya) [reply src/components/Steps/index.tsx:13]: @SebasEC96 i tested and work, both are options for order the array asc, i can commit your option if you like

Comment (SebasEC96) [reply src/components/Steps/index.tsx:13]: Are we sure that the array is not coming ordered ascendent from de sanity api?

Comment (Fedeya) [reply src/components/Steps/index.tsx:13]: the query don't have an order in the steps field

Comment (Fedeya) [reply src/components/Steps/index.tsx:13]: the order probably is the position in the array, that can be modified in the sanity

Comment (SebasEC96) [reply src/components/Steps/index.tsx:13]: Choose the option you like the most and we merge it 😛

**Review by SebasEC96 — APPROVED** [review]
LGTM

**Pull request merged**